README.md

```diff
@@ -1,17 +1,74 @@
 # Rapid Experiments Shared ![CircleCI](https://img.shields.io/circleci/build/github/mozilla/rapid-experiments-shared)
 
-This is a place to define data and schemas used across the rapid experiments
-program.
+This is a place to define data and schemas used across the rapid experiments program.
 
-Any data that moves between systems should have a schema defined here. Schemas
-are defined using Typescript, and automatically converted to JSON Schema. Any
-data that needs to be re-used by multiple systems should be stored here to be
-shared.
+Any data that moves between systems should have a schema defined here. Schemas are defined using
+Typescript, and automatically converted to JSON Schema. Any data that needs to be re-used by
+multiple systems should be stored here to be shared.
+
+## Using the schemas
+
+### Javascript
+
+Install this package
+
+```shell
+$ npm install --save @mozilla/rapid-experiments-shared
+```
+
+Import it.
+
+```js
+import mers from "@mozilla/rapid-experiments-shared";
+```
+
+JSON Schemas can be accessed in `mers.schemas` from JS, or in
```

Comment:
I think it would be sufficient to show an example of `import {schemas} from ...`